docs/general/testing/web/index.md

```diff
@@ -26,3 +26,57 @@ Each PR of [Jellyfin Web](https://github.com/jellyfin/jellyfin-web) and [Jellyfi
 :::caution
 Using a development version of clients may lead to data corruption or loss on the server. Please use a dedicated test server and make regular backups of [Jellyfin files](/docs/general/administration/configuration#server-paths).
 :::
+
+## Testing from source code
+
+Jellyfin web clients can be hosted as a standalone application without being associated with a Jellyfin server. In that case, a separate Jellyfin server will need to be prepared.
```

Comment:
```suggestion
Jellyfin web clients can be hosted as a standalone application without being associated with a Jellyfin server. In that case, a separate Jellyfin server will be needed.
```
I'm not sure that second sentence it's necessary as well. The first one is clear enough and I feel like the second one could be misinterpreted, leading people to think that at least 2 servers are needed somehow?